oaknational/Oak-Web-Application
# feat(curric): add routing

## Description

- Moved curriculum info page
- Added routing from phase subject picker to curriculum info page

## How to test

1. Go to [/beta/teachers/curriculum](https://deploy-preview-1777--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum)
2. Select a Subject, Phase and (if present) an examboard
3. You should navigate to the curriculum info page (with placeholder data)

## Checklist
